datadog/threadstats/base.py

```diff
@@ -28,10 +29,16 @@ def __init__(self, constant_tags=None):
 
         :param constant_tags: Tags to attach to every metric reported by this client
         :type constant_tags: list of strings
+
+        :envvar DOGSTATSD_TAGS: Tags to attach to every metric reported by ThreadStats client
+        :type constant_tags: list of strings
         """
         # Don't collect until start is called.
         self._disabled = True
-        self.constant_tags = constant_tags
+        self.env_tags = [tag for tag in os.environ.get('DOGSTATSD_TAGS', '').split(',') if tag]
```

Comment:
Same as for `datadog/dogstatsd/base.py`: I reckon there is no benefit to make it an instance variable.